groups/groups.yaml

```diff
@@ -1278,6 +1277,7 @@ groups:
       - riaan@ii.coop
       - stephen@ii.coop
       - zz@ii.coop
+      - asn-etl@k8s-infra-ii-sandbox.iam.gserviceaccount.com
```

Comment:
@bernokl I'm not sure to understand why you need to add this SA to this google group. What is the particularity of this SA ?

Reply:
@ameukam it will be our reporting user that is intended to run the ash-ip-lookup job on prow. Is there a more appropriate way for a job runner to get permissions to the logs?

Reply:
I would rather we keep service accounts out of groups.  I can't recall the specific example right now, but it doesn't always work.  I removed all previous SA memberships in https://github.com/kubernetes/k8s.io/pull/2010, specifically https://github.com/kubernetes/k8s.io/pull/2010/commits/9ebc221608f429d41db4d18bf8dd6d9f9ec2a21a

Bind the service account directly to what is needed

Reply:
@bernokl My suggestion would be look at https://cloud.google.com/iam/docs/understanding-roles to identity what permissions you need for this SA and add the binding as a Terraform code in https://github.com/kubernetes/k8s.io/tree/main/infra/gcp/clusters/projects/k8s-infra-ii-sandbox. 

Ultimately, every resource used to do PII analysis will be moved to https://github.com/kubernetes/k8s.io/tree/main/infra/gcp/clusters/projects/k8s-infra-public-pii. See : https://github.com/kubernetes/k8s.io/issues/1968.